agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java

```diff
@@ -741,23 +786,26 @@ private void exportRequest(SpanData span) {
       telemetry.getTags().put(ContextTagKeys.AI_LOCATION_IP.toString(), locationIp);
     }
 
-    data.setSource(getSource(attributes));
-
-    if (isAzureQueue(attributes)) {
```

Comment:
is there a reason why this check is removed?

Reply:
it's not needed, since only azure queue uses the `AZURE_SDK_ENQUEUED_TIME` attribute that's checked below